lib/future.coffee

```diff
@@ -13,7 +13,7 @@ emit = ($item, item) ->
   if transport = item.create?.source?.transport
     $item.append """<br><button class="transport" data-slug=#{item.slug}>create</button> transport from #{transport}"""
     $item.append "<p class=caption> unavailable</p>"
-    $.get 'http://localhost:4020', ->
+    $.get '//localhost:4020', ->
```

Comment:
https://github.com/fedwiki/wiki-client/pull/155 that introduced this code includes a picture.